registration/include/pcl/registration/impl/transformation_estimation_eig.hpp

```diff
@@ -0,0 +1,238 @@
+/*

```

Comment:
Using the SPDX header is better for brevity, specially for new files. A sample of that is present in my PR: https://github.com/PointCloudLibrary/pcl/pull/3890/files#diff-90b0c97382a0fee3beab051ccc399f00R1